amethyst_core/src/system_ext.rs

```diff
@@ -1,140 +0,0 @@
-//! Extension system utilities.
```

Comment:
Why was this file deleted? It appears to be related to functionality that we provided on top of Specs, does Legion provide it natively?

Reply:
Ah yes, I forgot about this. I'm not sure how hard would it be to implement pausable systems in legion, but definitely not as simple. I will add a todo for this to the legion project.

Reply:
Once that ToDo is added, this can be resolved.

Reply:
Eh, actually, not going to block the PR on it

Reply:
Added it already 👍